src/components/ManageLoginData.jsx

```diff
@@ -0,0 +1,19 @@
+import React,{useState} from "react";
+import Login from "./Login";
+
+function ManageLoginData () {
+    const [login,setLogin] = useState({
+        username : "",
+        password : "",
+    });
+
+    function onLoginChange (event){
+        const updateDetails = {...login, [event.target.name]: event.target.value};
+        setLogin(updateDetails);
+    }
+
+    return (
+            <Login login={login} onLoginChange={onLoginChange} />
```

Comment:
What is `login`? Passing in a property called `login` to a component called login is ambiguous. Please rename the `login` property to something more indicative of what the variable actually is

Reply:
@Stanley-Maraba you missed this comment. Please be careful when resolving comments yourself